Automation/2_2_1_ValmisteluMain.robot

```diff
@@ -1,21 +1,35 @@
-Documentation       Testataan dokumentaatiota
-'''                 testi 
 
 *** Settings ***
+Library     SeleniumLibrary
+Library     OperatingSystem
 
-Resource    ../Resources/2_2_1_Valmistelu.resource
+Resource    ../Resources/2_2_1_2_Osallistujayhteenveto.resource
 *** Variables ***
-${URL}    https://moodle.eoppimispalvelut.fi/
+*** Test Cases ***
+#    2_2_1_1 Kirjaudu moodleen
+#    2_2_1_2 Hae kurssin osallistujayhteenveto    (RYHMÄ 10 SAMI TEKEE)
```

Comment:
Jotta tämä pysyisi siistinä, niin nuo avainsanat pitäis siirtää 2_2_1_2 jonka 6 rivillä otat käyttöön. Siellä sitten esitellään tuo 10 rivin avainsana, nin että kun tuon 10 rivin ottaa pois kommenteista, ei näy virhettä.